images.CI/linux-and-win/lint_ubuntu.ps1

```diff
@@ -0,0 +1,34 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
+    )
+     $WrongScript = New-Object System.Collections.Generic.List[System.Object]
```

Comment:
Why can't we just use PS array?
```
$array = @()
$array += $el1
$array += $el2
```

Reply:
Yes, we can, I have changed this. Thank you.